src/start_request.cpp

```diff
@@ -22,14 +22,13 @@
 #include <string>
 #include <iostream>
 
-#include "psen_scan_v2/tenth_degree_conversion.h"
+#include "psen_scan_v2/angle_conversions.h"
 #include "psen_scan_v2/start_request.h"
-#include "psen_scan_v2/degree_to_rad.h"
 #include "psen_scan_v2/raw_processing.h"
 
 namespace psen_scan_v2
 {
-static constexpr double MASTER_RESOLUTION_RAD{ degreeToRad(1.) };
+static constexpr double MASTER_RESOLUTION_RAD{ degreeToRadian(1.) };
```

Comment:
The resolution of the master is 0.1 degree, isn't it?

Reply:
Where is the resolution defined?

Reply:
Ok now I get it. Yes we could set 0.1 here I think...